packages/aws-cdk/CONTRIBUTING.md

```diff
@@ -19,6 +19,15 @@ Run:
 npm run integ-cli
 ```
 
+In addition, you can run CLI integration compatibility tests. These run integration tests of the previously published version. They make sure you don't introduce changes that break any existing functionality.
```

Comment:
Probably should be moved to the second PR

Reply:
Whoops.